docs/user-guide/address-security-requirements.md

```diff
@@ -2,25 +2,15 @@
 
 As a ... (Identify the role/roles who are addressing these requirements. We should include an **Important:** note to inform readers in which use case they are likely to need permissions set by their security administrator.)
 
-Zowe implements comprehensive measures to secure mainframe services and data resources in transition and in rest:
-
-- Digital certificates are used by Zowe to facilitate secure electronic communication and data exchange between people, systems, and devices online.
-- User identity is authenticated through modern authentication methods such as OIDC/Oauth2, Multi-Factor Authentication (MFA), JWT, or Personal Access Token (PAT).
-- User access is authorized by System Authorization Facility (SAF) / External Security Manager (ESM).
+There are two parts of configuring security:
+* Configuring Zowe security
+* Extending Zowe
```

Comment:
Extending sounds as if you need to read it only when you are extending Zowe. 

What about something like: Additional Security Concepts?